backend/gn_module_monitoring/command/cmd.py

```diff
@@ -183,6 +184,29 @@ def cmd_install_monitoring_module(module_code):
     return
 
 
+@click.command("clean_existing_permissions")
+@click.argument("module_code", required=False, default="")
+@with_appcontext
+def cmd_clean_existing_permissions(module_code):
+    """
+    Nettoyage des permissions disponibles
+        Mise à jour des objets permissions associés au module
+        Transfert des droits pour les utilisateurs
+        Nettoyages des permissions et des permissions disponibles
+
+    """
```

Comment:
**Suggestion:** Add a docstring to `cmd_clean_existing_permissions` function [best practice]
```suggestion
187 +@click.command("clean_existing_permissions")
188 +@click.argument("module_code", required=False, default="")
189 +@with_appcontext
190 +def cmd_clean_existing_permissions(module_code):
191 +    """
192 +    Nettoyage des permissions disponibles
193 +    Mise à jour des objets permissions associés au module
194 +    Transfert des droits pour les utilisateurs
195 +    Nettoyages des permissions et des permissions disponibles
196 +
197 +    Args:
198 +        module_code (str, optional): The module code. Defaults to "".
199 +    """
```